src/models/ProjectUser/createProjectUser/index.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import type { ProjectUser, ProjectUserRole } from '@models/ProjectUser'
+import type { ValidationError } from '@types/ValidationError'
+import database from '@server/database'
+
+export type CreateProjectUserArgs = {
+  projectId: string,
+  userId: string,
+  role: ProjectUserRole
+}
+
+export type CreateProjectUserResponse = {
+  projectUser?: ProjectUser,
+  errors: Array<ValidationError>
+}
+
+const createProject = async ({
+  projectId,
+  userId,
+  role
+}: CreateProjectUserArgs): Promise<CreateProjectUserResponse> => {
+  const errors: Array<ValidationError> = []
+
+  if (errors.length) {
+    return { errors }
+  }
+
+  const savedProject: Array<ProjectUser> = await database('projectsUsers')
```

Comment:
Is this the correct pluralization for a relation table? I’m not sure? cc @PauuloG